src/utils/render-utils.ts

```diff
@@ -57,6 +57,7 @@ export function escapeTextForBrowser(text) {
 
 export function noop() {}
 
+// @ts-expect-error FIXME(implicitAny)
```

Comment:
couldn't find a type that worked but this is only used by the render-to-json files anyway